src/Application/Actions/EmailVerificationToken/Create.php

```diff
@@ -10,11 +10,44 @@
 use BigGive\Identity\Repository\PersonRepository;
 use Doctrine\ORM\EntityManagerInterface;
 use Laminas\Diactoros\Response\JsonResponse;
+use OpenApi\Annotations as OA;
 use Psr\Http\Message\ResponseInterface as Response;
 use Psr\Http\Message\ServerRequestInterface as Request;
 use Psr\Log\LoggerInterface;
 use Slim\Exception\HttpBadRequestException;
 
+/**
+ *  @OA\Post(
+ *     path="/v1/emailVerificationToken",
+ *     summary="Create a an email address verification token in advance of creating a donor account",
+ *     operationId="email_verification_token_create",
+ *     security={
+ *         {"captcha": {}}
+ *     },
+ *     @OA\RequestBody(
+ *         description="",
+ *         required=true,
+ *         @OA\JsonContent(
+ *              @OA\Property(property="email_address", type="string", example="fred@example.com"),
+ *        )
+ *     ),
+ *     @OA\Response(
+ *         response=201,
+ *         description="Token created and emailed to user, if they are NOT already registered exist, which the server
+ *          does not confirm or deny via http. If they are already registered then they are sent a message to remind
```

Comment:
stars in the middle of the description (at the start of lines) are not great - they render as bullet points in the openapi html output. In some other parts of this PR I used single line descriptions to avoid the issue. This was following examples in existing code e.g. https://github.com/thebiggive/identity/blob/00922f12123b11b8ee266d430d78c0a500f70ee5/src/Application/Actions/GetDonationFundsTransferInstructions.php#L37

Reply:
Yeah I remember this being a bit frustrating. I think the attribute usage of this library is better, although it might still have a similar problem with code indents – can't remember if that was still true with attributes or also with annotations.

Reply:
Right the stars issue wouldn't happen with attributes, and long lines wouldn't be much an an issue either since you can use concatenation inside attribute params.